workflow/rules/ref.smk

```diff
@@ -146,3 +146,18 @@ rule get_vep_plugins:
         "logs/vep/plugins.log",
     wrapper:
         "v3.3.5/bio/vep/plugins"
+
+
+# github link to the vg index for "xg" extension with the corresponding commit ID:
+# https://github.com/human-pangenomics/hpp_pangenome_resources/tree/201b532d11e8ba75a97db5005b98a780f42feeb6
+
+
+rule get_vg_pangenome:
+    output:
+        "resources/pangenome/hprc-v1.0-mc-grch38.xg",
```

Comment:
Rather name this analogously to the bwa index output, hence:
```suggestion
        f"{genome}.xg",
```

Reply:
renamed and link to download was added to config for exemplary purposes.